modules/gateway/nodes.go

```diff
@@ -133,9 +144,10 @@ func (g *Gateway) requestNodes(conn modules.PeerConn) error {
 		return err
 	}
 
+	const inbound = true
```

Comment:
If Go supported `const` on all types then I would definitely prefer defining everything as `const` unless it needs to be a variable. But since in practice only a few types can be `const`, we typically default to `var` except for certain top-level declarations.